### PR TITLE
Build branch logging SE in main build

### PR DESCRIPTION
**Pull request by pakrym**

### CODE REVIEW

File: src/Azure/AzureAppServices.HostingStartup/src/Microsoft.AspNetCore.AzureAppServices.HostingStartup.csproj [hunk 1]
```diff
@@ -7,6 +7,7 @@
     <TargetFrameworks>netcoreapp2.1;netcoreapp2.0;net461</TargetFrameworks>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <PackageTags>aspnetcore;azure;appservices</PackageTags>
+    <IsPackageInThisPatch>true</IsPackageInThisPatch>
```
Comment (pakrym) [inline src/Azure/AzureAppServices.HostingStartup/src/Microsoft.AspNetCore.AzureAppServices.HostingStartup.csproj]: This is to make sure everything passes in CI build.

Comment (natemcmaster) [reply src/Azure/AzureAppServices.HostingStartup/src/Microsoft.AspNetCore.AzureAppServices.HostingStartup.csproj]: Why was this necessary? Are we really releasing a patch of this package? If so, can you  list this package in eng/PatchConfig.props instead?

File: src/SiteExtensions/Sdk/SiteExtension.targets [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    <PackageReference Include="Microsoft.AspNetCore.App" Version="$(MicrosoftAspNetCoreAppPackageVersion)" IsImplicitlyDefined="true"/>
+    <PackageReference Include="%(HostingStartupPackageReference.Identity)" Version="%(HostingStartupPackageReference.Version)" />
+  </ItemGroup>
+</Project>
+      ]]>
+      </ManifestFileContents>
+    </PropertyGroup>
+
+    <ItemGroup Condition="'@(HostingStartupRuntimeStoreTargets->Count())' == '0'">
+      <HostingStartupRuntimeStoreTargets Include="netcoreapp3.0" Runtime="win7-x64" />
```
[the remark this answers is not in the document]
Comment (pakrym) [reply src/SiteExtensions/Sdk/SiteExtension.targets]: I'll clean up these usages, they are being overridden in other places.

**Comment by pakrym** [comment]
:up::date: this is ready to go.

File: src/Azure/AzureAppServices.HostingStartup/src/Microsoft.AspNetCore.AzureAppServices.HostingStartup.csproj (hunk 1 above)
Comment (natemcmaster) [reply src/Azure/AzureAppServices.HostingStartup/src/Microsoft.AspNetCore.AzureAppServices.HostingStartup.csproj]: Can you remove this before we merge?

**Review by natemcmaster — APPROVED** [review]

**Pull request merged**